.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
just curious- why this and not just a cargo fmt test? (i'm open to it, but i dont want style battles ;))

Reply:
This is so editors like vim and vscode can (with plugins) automatically set the correct settings. I usually add it to my projects. Want me to delete it?

I'll also run rustfmt!